mlflow/models/model.py

```diff
@@ -107,6 +127,10 @@ def model_uuid(self):
         """
         The ``model_uuid`` of the logged model,
         e.g., ``'39ca11813cfc46b09ab83972740b80ca'``.
+
+        :getter: [Legacy] Gets the model_uuid (run_id) of a logged model
+        :setter: [Legacy] Sets the model_uuid (run_id) of a model
```

Comment:
Similar question with above, _model_uuid should be immutable once model was logged ?